eo-runtime/src/test/eo/org/eolang/tuple-tests.eo

```diff
@@ -108,147 +113,89 @@
       [e]
         e > @
       nop
-    $.is
-      $.equal-to
-        "Can't #at(20) the 21th element of the tuple, there are just 4 of them"
-
-[] > concat-tuple-with-empty-tuple-1
-  * 1 2 3 > a
-  * > b
-  concat. > res
-    list a
-    b
-  assert-that > @
-    res.length
-    $.equal-to 3
-
-[] > concat-tuple-with-empty-tuple-2
-  * > a
-  * 1 2 3 > b
-  concat. > res
-    list a
-    b
-  assert-that > @
-    res.length
-    $.equal-to 3
-
-[] > concat-tuple-with-empty-tuple-3
-  * > a
-  * > b
-  concat. > res
-    list a
-    b
-  assert-that > @
-    res.length
-    $.equal-to 0
-
-[] > concat-tuples
-  * 1 2 3 > a
-  * 100 200 > b
-  concat. > res
-    list a
-    b
-  assert-that > @
-    plus.
-      res.at 2
-      res.at 4
-    $.equal-to 203
-
-[] > concat-three-tuples
-  * 0 1 2 > a
-  * 3 4 > b
-  * (* "five" "six") (* "siven") > c
-  concat. > res
-    list a
-    concat.
-      list b
-      c
-  assert-that > @
-    * (res.length) ((res.at 5).at 1)
-    $.equal-to
-      list
-        * 7 "six"
```

Comment:
@maxonfjvipon Why did you remove tests above?

Reply:
@Graur because the tests are not related to `eo-runtime`. Here `list.concat` object is tested, which is part of `eo-collections` repository. Moreover, we don't have any object in `eo-runtime` to replace `list.contac`